src/phases/execution/TimeSeriesSortOverlappingBucketsCommands.yml

```diff
@@ -0,0 +1,31 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  These are the phases used to measure performance of the bounded sorter for timeseries collections.
+
+BlockingSortCmdTemplate:
+  aggregate: {^Parameter: {Name: "coll", Default: Collection0}}
+  pipeline: [{$sort: {t: 1}}, {$_internalInhibitOptimization: {}}, {$skip: 1e10}]
+  cursor: {batchSize: {^Parameter: {Name: "batchSize", Default: 30000}}}
+  allowDiskUse: true
+
+BoundedSortCmdTemplate:
+  aggregate: {^Parameter: {Name: "bucketsColl", Default: system.buckets.Collection0}}
+  pipeline: [
+    {$_internalUnpackBucket: {
+      exclude: [],
+      timeField: "t",
+      metaField: "m",
+      bucketMaxSpanSeconds: 3600,
+      assumeNoMixedSchemaData: true
+    }},
+    {$_internalBoundedSort: {
+      sortKey: {t: 1},
+      bound: {base: "min"},
+    }},
+    {$_internalInhibitOptimization: {}},
+    {$skip: 1e10}
```

Comment:
Possibly dumb comment but you may want to do this as a limit query to simulate time to first result in addition. While I am happy we do see a marginal speed up in the non-degenerate case. It would be interesting to seem time to first result specifically because that is the case that we're optimizing for.

Reply:
Yea that's a good idea. I don't think that will be a difficult change either.